### PR TITLE
Change send email log line level

### PR DESCRIPTION
**Pull request by tej99**
# Motivation and Context

Currently the whole response from gov notify is being logged at
debug level. The response has senstive date (email address etc.)
which should not be logged, so I have changed the log lines to
extract specific information from the response. I have also changed
the send email log line from debug to info to be able to monitor
sending of emails in prod until we have more confidence around
action execution.

# What has changed

- Changed log lines to remove sensitive data
- Changed log level for send email response to info

# How to test?

# Links

### CODE REVIEW

**Comment by tej99** [comment]
[Determine time for email notifications to be sent per hour (Splunk) (1d)](https://trello.com/c/zUgNpKzw/145-determine-time-for-email-notifications-to-be-sent-per-hour-splunk-1d)

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# top-most EditorConfig file
```
Comment (NickGrantONS) [inline .editorconfig]: This file should probably be .gitignore'd

Comment (tej99) [reply .editorconfig]: I disagree, keeping this file means the coding style is maintained between different editors and IDEs.

File: .editorconfig [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
+end_of_line = lf
+
+# 2 space indentation
+[*.java]
```
Comment (NickGrantONS) [inline .editorconfig]: this is nowhere near complete - I think it's misleading. We use the [Google Java Style](https://google.github.io/styleguide/javaguide.html) which is pretty comprehensive.

Comment (tej99) [reply .editorconfig]: True that, removed 👍

**Pull request merged**